apps/test-urm04/src/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include "URM04Sensor.h"
+
+int main() {
+  // D2 - trigpin
+  // D1 - TX
+  // D0 - RX
+  URM04Sensor sensor(D2, D0, D1);
+  while (1) {
+    sensor.compute_distance();
+    sensor.print_distance();
```

Comment:
have a wait in here

Reply:
I already have a wait in the functions but Ill add another one here

Reply:
if i understand what you're doing, you try to slow down this loop iside the member functions.